grails-app/controllers/org/broadinstitute/orsp/UrlMappings.groovy

```diff
@@ -14,7 +14,9 @@ class UrlMappings {
         "/api/report/funding"(controller: 'report', action: "funding")
         "/"(view:"/index")
         '/api/swagger/**'(controller: 'api', action: 'swagger')
-
+        '/api/project'(resource: 'project')
+        '/api/files-helper/attach-document'(controller: 'fileHelper', action: 'attachDocument', method: 'POST')
+        '/api/newConsentGroup'(resource: 'newConsentGroup')
```

Comment:
On lines 18 and 19, we have two different styles of url here. Grails, by default, uses camel casing based on controller names (i.e. `ConsentGroupController` -> `/consentGroup`) which is not unreasonable. I actually like `kebab-case` urls so if we can try to be consisnt with that as we move forward, that would be my preference.